src/catalua.cpp

```diff
@@ -49,6 +49,9 @@ lua_State *lua_state = nullptr;
 // we know where to load files from.
 std::string lua_file_path = "";
 
+std::stringstream lua_output_stream;
+std::stringstream lua_error_stream;
```

Comment:
Out of curiosity, why are those generic in/out streams? Wouldn't a simple `std::ostream` be enough?